src/views/List.jsx

```diff
@@ -27,38 +30,62 @@ export function List({ data }) {
 		setSearchQuery('');
 	}
 
+	function handleNav() {
+		navigateTo('/add-item');
+	}
+
 	return (
-		<>
-			<p>
-				Hello from the <code>/list</code> page!
-			</p>
-			<form>
-				<label htmlFor="search-items">
-					Search Items:
-					<input
-						name="search-items"
-						id="search-items"
-						type="text"
-						placeholder="Search items"
-						value={searchQuery}
-						onChange={(e) => setSearchQuery(e.target.value)}
-					/>
-					{searchQuery ? (
-						<button type="button" onClick={handleClearSearchQuery}>
-							Clear search
-						</button>
+		<div className="list-container">
+			{loading ? (
+				<p>loading</p>
+			) : (
+				<>
+					{data.length >= 1 ? (
+						<>
+							<h2>Find what you're looking for!</h2>
+							<form>
+								<label htmlFor="search-items">
+									Search Items:{' '}
+									<input
+										name="search-items"
+										id="search-items"
+										type="text"
+										placeholder="Search items"
+										value={searchQuery}
+										onChange={(e) => setSearchQuery(e.target.value)}
+									/>
+									{searchQuery ? (
+										<>
+											{' '}
+											<button type="button" onClick={handleClearSearchQuery}>
+												Clear search
+											</button>
+										</>
+									) : (
+										''
+									)}
+								</label>
+							</form>
+						</>
 					) : (
-						''
-					)}
-				</label>
-			</form>
+						<>
+							<h2>
```

Comment:
Nice work on adding this and the logic for the conditional render on this page!

Reply:
💃  Thank you, Lindsey!